### PR TITLE
Homework #4

### PR DESCRIPTION
**Pull request by AndryNosach**

### CODE REVIEW

**Review by dinamio — COMMENTED** [review]
Good job! Can you add registration logic for this?

**Comment by AndryNosach** [comment]
would it be ok to save login/password in the file and then read from it?

**Comment by dinamio** [comment]
Yes, we can save in file, can save in memory, doesn't matter for now

**Comment by AndryNosach** [comment]
Commited new version with authirization.

File: Nosach/web-quiz-project/src/main/java/servlet/RegisterServlet.java [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    @Override
+    protected void doPost(HttpServletRequest req, HttpServletResponse resp) throws ServletException, IOException {
+
+        String login = req.getParameter("login");
+        String pass = req.getParameter("pass");
+        String email = req.getParameter("email");
+
+        try (FileWriter fw = new FileWriter(getServletContext().getRealPath("/resources/properties/users.properties"), true)) {
+
+            fw.write(login + "#" + pass + "#" + email + "\n");
```
Comment (dinamio) [inline Nosach/web-quiz-project/src/main/java/servlet/RegisterServlet.java:27]: Could be problem if login or pass contains #

**Review by dinamio — APPROVED** [review]
Looks good

File: Nosach/web-quiz-project/src/main/java/servlet/RegisterServlet.java (hunk 1 above)
Comment (AndryNosach) [reply Nosach/web-quiz-project/src/main/java/servlet/RegisterServlet.java:27]: yes, it would be problem.  will be fixed in the future when we'll use db

**Pull request merged**